content/blog/2023/06/22/2023-06-22-mirrors-jenkins-new-IP.adoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "New Public IPv4 for Jenkins Mirrors"
+tags:
+- infrastructure
+- mirrors
+- jenkins
+authors:
+- dduportal
+opengraph:
+  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
+links:
+  discourse: https://community.jenkins.io/t/new-public-ipv4-for-jenkins-mirrors/8137
+---
+
+The Jenkins project provides a download mirror infrastructure allowing to download Jenkins packages and plugins from a download server close to your location.
```

Comment:
```suggestion
The Jenkins project provides a download mirror allowing to download Jenkins packages and plugins from a download server close to your location.
```

Reply:
I would also remove the first "download" to avoid repetition:
```suggestion
The Jenkins project provides a mirror allowing to download Jenkins packages and plugins from a download server close to your location.
```